components/Header.tsx

```diff
@@ -19,6 +19,7 @@ export default function Header() {
       if (window.innerWidth >= 768) {
         // Tailwind's md breakpoint
         setIsMenuOpen(false);
+        document.body.classList.remove('menu-open'); // Remove the CSS class when the menu is closed
```

Comment:
is this needed? `setIsMenuOpen` should trigger our useEffect which should remove the class

Reply:
Totally correct. Forgot to move the original implementation as it stopped working. I'll make that change.